src/main/java/com/github/elenterius/biomancy/tileentity/DigesterTileEntity.java

```diff
@@ -252,7 +257,30 @@ else if (FluidUtil.getFluidContained(stack).isPresent()) {
 			}
 		}
 	}
-
+	
+	protected void tryToSuckWater(World world) {
+		int fluidAmount = stateData.fuelTank.getFluidAmount();
+		int maxFluidAmount = stateData.fuelTank.getCapacity();
+		if (fluidAmount <= maxFluidAmount-1000) {
+			Direction direction = getBlockState().get(DigesterBlock.FACING);
+			BlockPos blockPos = getPos().offset(direction);
+			Block neighbourBlock = world.getBlockState(blockPos).getBlock();
+			if (neighbourBlock == Blocks.WET_SPONGE) {
+				world.setBlockState(blockPos, Blocks.SPONGE.getDefaultState(), 3);
+				addFuelAmount(1000);
+			}
+		}
+	}
+	
+	protected void tryToInjectFluid(World world) {
+		Direction direction = getBlockState().get(DigesterBlock.FACING).getOpposite();
+		BlockPos blockPos = getPos().offset(direction);
+		LazyOptional<IFluidHandler> capability = FluidUtil.getFluidHandler(world, blockPos, Direction.DOWN);
```

Comment:
Direction shouldn't be DOWN. Most mods expect UP or horizontal axis for inserting things.